src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
```

Comment:
Don't hardcode some values that may change in the future in variable names (numbers etc.)
BAD EXAMPLE:
```js
const discountForThreeDays = 50;
```
GOOD EXAMPLE:
```js
const basicDiscount = 50;
```

Reply:
thank you, I actually forgot about checklist🙈